4001-5000/LIT4633Chronicle.xml

```diff
@@ -914,27 +929,34 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 </div>
                 
                 <div type="textpart" subtype="paragraph" n="50">
-                    <ab> 1ክፍል፡ ፲፭።2ሠረቀ፡ ጥቅምት፡ በዕለተ፡ እሑድ።3ወአመ፡ ፲ወ፪፡ ለጥቅምት፡ አዕረፈ፡ ቄስ፡ ሐፄ፡ ቀውስጦስ፡ 
-                        ዐቢይ1፡ ካህን።4ወእምድኅረ፡ ኀለፈ2፡ ጊዜ፡ ፱፡ ሰዓት፡ ጸውዖ፡ ንጉሥ፡ ለአባ፡ ዘወልደ፡ ማርያም፡ አበ፡ መጉና።5ወነበረ፡ 
-                        ዲበ፡ መንበረ፡ መንግሥት፡ በማዕከላይ፡ ግምብ።6ወነገሮ፡ ከመ፡ የኀሥሥ3፡ ጽድቀ፡ በሕጉ።7ወአስተጋብኦሙ፡ ለካህናት፡ 
+                    <ab>ክፍል፡ ፲፭።2ሠረቀ፡ ጥቅምት፡ በዕለተ፡ እሑድ።3ወአመ፡ ፲ወ፪፡ ለጥቅምት፡ አዕረፈ፡  
+                        <persName ref="PRS12795Qawestos"><roleName type="title">ቄስ፡ ሐፄ፡</roleName> ቀውስጦስ፡ </persName>
+                        ዐቢይ፡ ካህን።ወእምድኅረ፡ ኀለፈ፡ ጊዜ፡ ፱፡ ሰዓት፡ ጸውዖ፡ <persName ref="PRS2377Bakaffa"><roleName type="title">ንጉሥ፡</roleName> </persName>
+                        ለአባ፡ ዘወልደ፡ ማርያም፡ አበ፡ መጉና። ወነበረ፡ 
+                        ዲበ፡ መንበረ፡ መንግሥት፡ በማዕከላይ፡ ግምብ። ወነገሮ፡ ከመ፡ የኀሥሥ3፡ ጽድቀ፡ በሕጉ።7ወአስተጋብኦሙ፡ ለካህናት፡ 
```

Comment:
I don't understand the use of Arabic numerals here and throughout the record - are they sentence numbers? Why does 7 follow 3? If they are sentence numbers, why are they encoded as part of the edited text, without even spaces between them and the words around them?

Reply:
Here I have only encoded person, places, and other entities. I have not considered the Arabic numbers that refer to the footnotes, so I will delete all Arabic numerals